pkg/server/app/app.go

```diff
@@ -96,8 +97,14 @@ func (ops *AppOperations) Create(user *storage.User, app *App) error {
 		return teresa_errors.NewInternalServerError(err)
 	}
 
+	defer func() {
+		if Err != nil {
```

Comment:
Very good point but I guess the defer is only installed if me passed the stage of returning these 2 errors, what do you think?